apps/dav/lib/CalDAV/BirthdayService.php

```diff
@@ -273,6 +273,7 @@ public function buildDateFromContact(string $cardData,
 	public function resetForUser(string $user):void {
 		$principal = 'principals/users/'.$user;
 		$calendar = $this->calDavBackEnd->getCalendarByUri($principal, self::BIRTHDAY_CALENDAR_URI);
+		if (!$calendar) return; // The user's birthday calendar doesn't exist
```

Comment:
```suggestion
		if (!$calendar) {
			return;
		}
```

we always use brackets

Reply:
I thought cs-fixer would have caught that :thinking: